orangecontrib/educational/widgets/owkmeans.py

```diff
@@ -1,15 +1,17 @@
+import numpy as np
+from os import path
+from itertools import chain
+import time
+
+from PyQt4.QtCore import pyqtSlot, QThread, SIGNAL, Qt
+from PyQt4.QtGui import QSizePolicy
+
```

Comment:
The [rule](https://www.python.org/dev/peps/pep-0008/#imports) is to first import the built-ins, then other libraries, then your modules.